dev-embedded/xa/metadata.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="project">
+    <email>ckaiser@floodgap.com</email>
```

Comment:
seems like we need to use a gentoo maintainer here, and not the original software developer.

Reply:
right, but someone must raise hand here :smile: , maybe app-emulation/vice maintainer, since it's really small piece which only vice uses at this moment?

Reply:
Right, just stick it under games@gentoo.org.

Reply:
done